Scripts/TF/TransformPublisher.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Ros_CSharp;
+using tf.net;
+
+public class TransformPublisher : MonoBehaviour {
+
+    public ROSCore rosmaster;
+
+    private NodeHandle nh = null;
+    private Publisher<Messages.tf.tfMessage> tfPub;
+
+    public GameObject trackedObject;
+
+    public string frame_id;
+    public string child_frame_id;
+
+    public bool using_gazebo;
+
+
+    // Use this for initialization
+    void Start () {
+        NodeHandle nh = rosmaster.getNodeHandle();
+        tfPub = nh.advertise<Messages.tf.tfMessage>("/tf", 100);
+
+    }
+	
+	// Update is called once per frame
+	void Update () {
```

Comment:
Whitespace! :-P

Reply:
fixed. 

And I have no idea if those dll will work on other platforms, I assume not, unless Unity does some very fancy stuff. I'll test it later though, I am curious.